csclient/csclient_test.go

```diff
@@ -12,7 +12,7 @@ import (
 	"io/ioutil"
 	"net/http"
 	"net/http/httptest"
-	"net/url"
+	neturl "net/url"
```

Comment:
Because "url" is used as identifier in the test we changed (and also lots of the other tests), so better to change the package identifier, I think.